labs/b2.md

```diff
@@ -122,6 +131,8 @@ Note: This behavior is from bash and some other shells that implement it.
  - Detach from a session with `Ctrl-b d` (press d after releasing `Ctrl-b`)
  - Split into 2 panes with `Ctrl-b %` (vertical) and `Ctrl-b "` (horizontal)
  - Swap current pane with `Ctrl-b o`
+ - Create new window with `Ctrl-b c`
+ - Go to next window with `Ctrl-b n` 
```

Comment:
Also previous with `Ctrl-b p`, but I think that's about all they'd need to use the windows honestly, they're pretty simple which is nice.